src/content-cli-push.ts

```diff
@@ -91,6 +93,12 @@ class Push {
             .option("--packageManager", "Upload widget to package manager (deprecated)") // Deprecated
             .action(async cmd => {
                 await new WidgetCommand().pushWidget(cmd.profile, !!cmd.tenantIndependent, !!cmd.userSpecific);
+                await new WidgetSourcemapsCommand().pushWidget(
```

Comment:
I think we should add a flag for this and not upload it always. And also, maybe we should make it dependent if `--tenantIndependent` is being used as well, because idk if it makes that much sense to push the source maps without that flag being true.

Reply:
we don't push if the environment variables are not set. Check widget-sourcemaps.manager.ts `pushWidgetSourcemaps` method.

Reply:
Ah okay I haven't seen that. Personally I think it's nicer to have a separate flag rather than making it dependent on the environment variables, because it's not clear that whenever you use env variables => it pushes something to datadog. Additionally, that still doesn't ensure that the source maps are not pushed when we push only to a specific team or a user.

Reply:
Yeah as you wish, I don't have a good debate. I will add that as a flag.